src/ui/components/todomvc-app/component.ts

```diff
@@ -1,24 +1,22 @@
 import Component, { tracked } from "@glimmer/component";
 import Navigo from 'navigo';
 import TodoStore from '../../../utils/todo-store';
-import Todo from '../utils/todo';
+import Todo from '../../../utils/todo';
+import { ENTER } from '../../../utils/keys';
 
 const router = new Navigo(null, true);
 
-
-
 export default class TodoMVCApp extends Component {
-
-  storage: TodoStore;
+  todoStore = new TodoStore();
 
   @tracked todos: Todo[] = [];
```

Comment:
Mostly for my edification: is this safe in a Glimmer component? In a plain JavaScript object, this would result in every instance of `TodoMVCApp` sharing the same `todos` Array.